dg-core/src/main/java/org/finra/datagenerator/consumer/DataFormatter.java

```diff
@@ -23,8 +23,9 @@
  * Wrapper for search results.
  *
  * Created by RobbinBr on 5/18/2014.
+ * Updated by Mauricio Silva on 4/24/2015
  */
-public class DataPipe {
+public class DataFormatter {
```

Comment:
This probably needs to be names backed to DataPipe, since it no longer does any formatting.